config/views/static.py

```diff
@@ -0,0 +1,145 @@
+from ds_caselaw_utils import courts
+
+from .template_view_with_context import TemplateViewWithContext
```

Comment:
Is it worth TemplateViewWithContext handling the feedback_survey_type context item, based on a value in the subclass (if it exists?)